src/genericCmps/dropdown/styles.ts

```diff
@@ -0,0 +1,78 @@
+import colors from "../../assets/constants/colors";
+import fonts from "../../assets/constants/fonts";
+import paddings from "../../assets/constants/paddings";
+import typography from "../../assets/constants/typography";
+import { SPACING } from "../../assets/constants/spacings";
+import { WIDTHS, HEIGHTS, CURSORS } from "../../assets/constants/dimensions";
+import { ALIGNMENTS, JUSTIFY_CONTENT } from "../../assets/constants/flex";
+
+export const dropdownStyles = {
+  container: {
+    position: "relative",
+    width: WIDTHS.fitContent,
```

Comment:
we dont need this const

Reply:
you mean the whole dropdownStyles or the container?

Reply:
the WIDTHS const